src/mbed_tools/cli/project_management.py

```diff
@@ -74,17 +74,15 @@ def libs(path: str) -> None:
     click.echo(tabulate.tabulate(table, headers=headers))
 
     if lib_data["unresolved"]:
-        click.echo(
-            "\nUnresolved libraries detected. Please run the `checkout` command to download library source code."
-        )
+        click.echo("\nUnresolved libraries detected. Please run the `deploy` command to download library source code.")
 
 
 @click.command()
 @click.argument("path", type=click.Path(), default=os.getcwd())
 @click.option(
-    "--force", "-f", is_flag=True, show_default=True, help="Force checkout, overwrites local uncommitted changes."
+    "--force", "-f", is_flag=True, show_default=True, help="Force deploy, overwrites local uncommitted changes."
```

Comment:
I think the help should say something like "Forces all library repositories to be checked out at the commit specified in the .lib file, overwrites local changes."